### PR TITLE
Removes forgotten player references

### PR DESCRIPTION
**Pull request by ArcaneDefence**
## About The Pull Request
Removes a player reference in favor of a generic name created on the spot for this replacement

## Why It's Good For The Game
Further maintains standard that player references aren't allowed

:cl:
fix: Removes more player references that were forgotten.
/:cl:

### CODE REVIEW

**Comment by moocowswag** [comment]
Do i have permission to summon the swarm

**Comment by zxaber** [comment]
Cuban Pete is hardly forgotten when admins play the iconic song after a traitor multi-maxcaps.

**Comment by Cheshify** [comment]
> fix
> no "why it's good"
> doesn't follow template
Begone

File: strings/wanted_message.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
 		"an NT CentCom uniform": 50,
 		"bluespace crystals": 50,
 		"clown tears": 5,
-		"Cuban Pete": 1,
+		"Spicy Steve": 1,
```
Comment (san7890) [inline strings/wanted_message.json:43]: this should be alphabetized

**Review by san7890 — CHANGES_REQUESTED** [review]

**Comment by san7890** [comment]
also you must fill out the full PR template

**Comment by AverageUser67** [comment]
the dark ages

**Comment by BeeSting12** [comment]
Cuban Pete may have been a player, but it also references a movie

https://www.youtube.com/watch?v=A8VqdhNnwdY

File: _maps/RandomRuins/SpaceRuins/oldAIsat.dmm [hunk 2]
```diff
@@ -126,7 +126,7 @@
 /area/ruin/space/tcommsat_oldaisat)
 "aE" = (
 /obj/item/food/meat/slab/synthmeat{
-	name = "Cuban Pete-Meat"
+	name = "Spicy Steve-Meat"
```
Comment (CRITAWAKETS) [inline _maps/RandomRuins/SpaceRuins/oldAIsat.dmm:129]:
```suggestion
	name = "Stinky Steve-Meat"
```
im sorry

**Pull request closed without merging**